gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/proc/ReevaluateDagProc.java

```diff
@@ -167,9 +160,5 @@ private void onJobFinish(DagManagementStateStore dagManagementStateStore, Dag.Da
       default:
         log.warn("It should not reach here. Job status {} is unexpected.", executionStatus);
     }
-
-    // Checkpoint the dag state, it should have an updated value of dag fields
-    dagManagementStateStore.checkpointDag(dag);
```

Comment:
is this removed bc the dag was not updated in this method?

Reply:
yes. only setMessage and setFlowEvent are called, which are anyway not persisted anyway